topics/transcriptomics/tutorials/srna/tours/tour.yaml

```diff
@@ -0,0 +1,771 @@
+      <b>Miscellaneous Manipulation Type</b> parameter tells the tool to remove
+      these sequences. What remains are sequences of length 20-22nt. We will now
+      repeat these steps to identify 23-29nt piRNA sequences.
+    backdrop: true
+  - title: Extract subclasses
+    element: '#tool-search-query'
+    content: Search for Manipulate FASTQ  tool
+    placement: right
+    textinsert: Manipulate FASTQ
+  - title: Extract subclasses
+    element: '#tool-search'
+    content: Click on the "Manipulate FASTQ" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastq_manipulation%2Ffastq_manipulation%2F1.0.1"]
+  - title: Extract subclasses
+    element: '#tool-search'
+    content: >-
+      Execute the tool on each collection of non-r/miRNA reads to identify
+      piRNAs (23-29nt) using the following parameters <ul> <li><b>FASTQ
+      File:</b> Click the “Dataset collection” tab and then select the blank
+      RNAi sRNA-seq dataset of non-r/miRNA FASTQ files</li> <li><b>Match
+      Reads:</b> Click “Insert Match Reads”</li> <li><b>Match Reads by:</b> Set
+      to “Sequence Content”</li> <li><b>Match by:</b> ^.{12,22}$ ^.{30,50}$</li>
+      <li><b>Manipulate Reads:</b> Click “Insert Manipulate Reads”</li>
+      <li><b>Manipulate Reads on:</b> Set to “Miscellaneous actions”</li>
+      <li><b>Miscellaneous Manipulation Type:</b> Set to “Remove Read”</li>
+      </ul>
+    position: left
+  - title: Extract subclasses
+    content: Repeat for the Symplekin RNAi dataset collection.
+    backdrop: true
+  - title: Extract subclasses
+    content: >-
+      Rename each resulting dataset collection something meaningful (i.e. “blank
+      RNAi - siRNA reads (23-29nt)”)
+    backdrop: true
+  - title: Extract subclasses
+    element: '#tool-search-query'
+    content: Search for FastQC  tool
+    placement: right
+    textinsert: FastQC
+  - title: Extract subclasses
+    element: '#tool-search'
+    content: Click on the "FastQC" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fdevteam%2Ffastqc%2Ffastqc%2F0.69"]
+  - title: Extract subclasses
+    element: '#tool-search'
+    content: >-
+      Run FastQC on each collection of siRNA and piRNA read files to confirm the
+      correct read lengths.
+    position: left
+  - title: Extract subclasses
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      Inspect the outputs. You should notice that we have ~32k reads that are
+      20-22nt long in our Reads_20-22_nt_length_siRNAs file and ~16k reads that
+      are 23-29nt long in our Reads_23-29_nt_length_piRNAs file, as expected.
+      Given that we had ~268k trimmed reads in the Blank RNAi replicate 3 file,
+      siRNAs represent ~12.1% of the reads and piRNAs represent ~6.2% of the
+      reads in this experiment.
```

Comment:
Here we could mention that there is no distinct peak in the piRNA range and refer to a publication where the reader can compare this to what a real piRNA profile looks like.

Reply:
@mvdbeek do you have an example of such publication?

Reply:
http://dx.doi.org/10.1016/j.cell.2009.03.040
Fig S1 is nice in that regard. Surprisingly few publications show that actually full size-distribution.